client/src/components/applicants/ApplicationStages.js

```diff
@@ -2,75 +2,57 @@ import React, { Component } from 'react'
 import { connect } from 'react-redux'
 import PropTypes from 'prop-types'
 import Applicant from './Applicant'
+import { Droppable } from 'react-beautiful-dnd'
 
 import * as actions from '../../redux/actions/actions'
 
 export class ApplicationStages extends Component {
-  constructor(props) {
-    super(props)
-    this.state = {
-      isOver: false
-    }
-  }
 
-  onDrag = (event, applicant) => {
-    this.props.onDrag(event, applicant)
-  }
-
-  onDragOver = (event) => {
-    if (!this.props.adminView) {
-      this.setState({ isOver: true })
-      event.preventDefault()
+  componentDidUpdate(nProps) {
+    if (nProps.stage.applicants !== this.props.stage.applicants) {
+      const contentDiv = document.getElementById('application-stage__content')
```

Comment:
The first parameter of componentDidUpdate is *previous props*, so if nProps stands for new props it's perhaps a bit misleading. But it works either way